jina/executors/indexers/dbms/keyvalue.py

```diff
@@ -14,7 +14,7 @@ def _get_generator(
         self, ids: List[str]
     ) -> Generator[Tuple[str, np.array, bytes], None, None]:
         for id_ in ids:
-            vecs_metas_list_bytes = super()._query(id_)
```

Comment:
we can add a `TODO` here to review if with new interface this could be done already using the batch. Not to change now

Reply:
I guess you could do
```
       vecs_metas_list_bytes=     vecs_metas_list_bytes = super()._query(ids)
       for k, id_ in enumerate(ids):
            vec, meta = pickle.loads(vecs_metas_list_bytes[k])
            yield id_, vec, meta
```
but the pickle will still work with a single element